docs/embeddings.md

```diff
@@ -135,22 +171,35 @@ openai_ef = embedding_functions.OpenAIEmbeddingFunction(
 <TabItem value="js" label="JavaScript">
 
 ```javascript
-const {OpenAIEmbeddingFunction} = require('chromadb');
-const embedder = new OpenAIEmbeddingFunction({openai_api_key: "apiKey"})
-
-// use directly 
-const embeddings = embedder.generate(["document1","document2"])
+const { OpenAIEmbeddingFunction } = require('chromadb');
+const embedder = new OpenAIEmbeddingFunction({ openai_api_key: 'apiKey' });
 
 // pass documents to query for .add and .query
-const collection = await client.createCollection({name: "name", embeddingFunction: embedder})
-const collection = await client.getCollection({name: "name", embeddingFunction: embedder})
+const collection = await client.createCollection({
+	name: 'name',
+	embeddingFunction: embedder,
+});
+const collection = await client.getCollection({
+	name: 'name',
+	embeddingFunction: embedder,
+});
+
+// use directly
+const embeddings = await embedder.generate(['document1', 'document2']);
+
+// Add documents to the collection along with their associated embeddings.
+// Chroma will store these documents without generating its own embeddings.
+await collection.add({
```

Comment:
nit: Same comment as for Cohere; let's consider if an additional example without `embeddings` can be beneficial here.